jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/actions/ShowLogsAround.kt

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.actions
+
+import com.intellij.icons.AllIcons
+import com.intellij.openapi.actionSystem.ActionGroup
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.LangDataKeys
+import com.intellij.openapi.project.DumbAware
+import com.intellij.ui.table.TableView
+import software.amazon.awssdk.services.cloudwatchlogs.model.OutputLogEvent
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.CloudWatchLogWindow
+import software.aws.toolkits.resources.message
+
+class ShowLogsAroundGroup(
+    private val logGroup: String,
+    private val logStream: String,
+    private val treeTable: TableView<OutputLogEvent>
+) : ActionGroup(message("cloudwatch.logs.show_logs_around"), null, AllIcons.Ide.Link), DumbAware {
+    init {
+        isPopup = true
+    }
+
+    override fun getChildren(e: AnActionEvent?): Array<AnAction> = arrayOf(
+        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.one_minute"), 60 * 1000),
+        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.five_minutes"), 5 * 60 * 1000),
+        ShowLogsAround(logGroup, logStream, treeTable, message("general.time.ten_minutes"), 10 * 60 * 1000)
+    )
+}
+
+private class ShowLogsAround(
+    private val logGroup: String,
+    private val logStream: String,
+    private val treeTable: TableView<OutputLogEvent>,
+    time: String,
+    private val durationMilliseconds: Long
+) :
+    AnAction(time, null, null), DumbAware {
+    override fun actionPerformed(e: AnActionEvent) {
+        val project = e.getRequiredData(LangDataKeys.PROJECT)
```

Comment:
unrelated, but it appears that: `LangDataKeys.PROJECT`, `CommonDataKeys.PROJECT`, `PlatformDataKeys.PROJECT` are all being used....

Reply:
Oh dear, `PlatformDataKeys.PROJECT` makes the most sense to me so changed it to use that